examples/index.html

```diff
@@ -98,6 +98,21 @@ <h1>Image Sequencer</h1>
   </div>
     <button class="btn btn-primary btn-lg" name="save-sequence" id="save-seq">Save Sequence</button>
 </div>
+  <!--the newly added code is here-->
+<br>
+  <footer class="page-footer font-small" style="margin:auto;background-color: lightgray;position: relative">
+      <a href="https://publiclab.org/" style="text-decoration: none;color:dimgray;position: absolute;left: 30px">   home</a><br>
```

Comment:
Please remove redundant spaces  ```  home```   ```  about```  ```  signup``` likewise from 104 to 114  indentations as i don't want to highlight each codeline..So please make the changes in your whole code where you just need to remove ```reduntant ``` spaces.